packages/client/src/api/oAuth/types.ts

```diff
@@ -0,0 +1,11 @@
+export type OauthSignInRequest = {
+    // User code from Yandex
+    code: string
+
+    // Redirect uri that you are using for oauth
+    redirect_uri: string
+}
+
+export type ClientID = {
```

Comment:
Объект содержит `clientId`, но не является им. Тут нужно либо переименовать полностью либо хотя бы так

```suggestion
export type ClientIdDTO = {
```